### PR TITLE
feat(gatsby-theme-bodiless): Inject canonical preset to gatsby image node.

### PR DESCRIPTION
**Pull request by wodenx**
## Changes

Inject the image preset as defined in code to the gatsby image node, making it available
to the image logger for comparison with atual data.

This makes it unnecessary to pass the preset explicitly to `withGatsbyImageLogger`.

### CODE REVIEW

**Review by marcopagliarulo — APPROVED** [review]

File: packages/gatsby-theme-bodiless/src/dist/GatsbyImage/withGatsbyImageLogger.tsx [hunk 1]
```diff
@@ -24,9 +24,11 @@ type Props = {
 
 const withGatsbyImageLogger = (preset: GatsbyImagePresets): Token => Component => {
```
Comment (marcopagliarulo) [inline packages/gatsby-theme-bodiless/src/dist/GatsbyImage/withGatsbyImageLogger.tsx:25]: preset should be optional to allow the usage of canonicalPreset

**Review by marcopagliarulo — CHANGES_REQUESTED** [review]

File: packages/gatsby-theme-bodiless/src/dist/GatsbyImage/withGatsbyImageLogger.tsx (hunk 1 above)
Comment (wodenx) [reply packages/gatsby-theme-bodiless/src/dist/GatsbyImage/withGatsbyImageLogger.tsx:25]: yes - please update.

Comment (marcopagliarulo) [reply packages/gatsby-theme-bodiless/src/dist/GatsbyImage/withGatsbyImageLogger.tsx:25]: I cannot update, the branch is on your fork, I am not allowed to push

Comment (wodenx) [inline packages/gatsby-theme-bodiless/src/dist/GatsbyImage/withGatsbyImageLogger.tsx:25]:
```suggestion
const withGatsbyImageLogger = (preset?: GatsbyImagePresets): Token => Component => {
```

**Review by marcopagliarulo — APPROVED** [review]

**Review by beliayeu — APPROVED** [review]

**Pull request merged**